lua/gitlinker/git.lua

```diff
@@ -1,97 +1,64 @@
 local M = {}
 
 local job = require("plenary.job")
+local path = require("plenary.path")
 
-local function get_remotes()
-  local remotes
-  local p = job:new({ command = "git", args = { "remote" } })
+-- wrap the git command to do the rigth thing alwasy
+-- @args: can be any git subcommand
+local function git(args)
+  local o = { '-C', tostring(path:new(vim.api.nvim_buf_get_name(0)):parent()) }
+  vim.list_extend(o, args)
```

Comment:
sorry I'm picky with variable names but `o` needs to be renamed to something like `scope_args`. Either that or you can just do:

```suggestion
  vim.list_extend(args, { '-C', tostring(path:new(vim.api.nvim_buf_get_name(0)):parent()) })
```